### PR TITLE
Add links explorer

### PR DESCRIPTION
**Pull request by anxolin**
* Review all links of the App
* Create a generic `<LinkExplorer>` component that will show the label and link to the right Explorer. It can be either GP explorer, Etherscan or Blockscout
* Adapted the Popup (toast)
* Adapted the confirmation modal
* Adapter the Account

Some examples:

Screenshot at Feb 26 14-27-26
Screenshot at Feb 26 14-26-33

Screenshot at Feb 26 14-27-40

Screenshot at Feb 26 14-28-39

Screenshot at Feb 26 14-29-09
Screenshot at Feb 26 14-29-31

### CODE REVIEW

File: src/custom/components/AccountDetails/index.tsx [hunk 1]
```diff
@@ -207,7 +208,7 @@ export default function AccountDetails({
                             href={getEtherscanLink(chainId, account, 'address')}
                           >
                             <LinkIcon size={16} />
-                            <span style={{ marginLeft: '4px' }}>View on Etherscan</span>
+                            <span style={{ marginLeft: '4px' }}>{explorerLabel}</span>
```
Comment (W3stside) [inline src/custom/components/AccountDetails/index.tsx:211]: nit: make a styled comp and dont use inline

File: src/custom/components/ExplorerLink/index.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+export function ExplorerLink(props: Props) {
+  const { id, label, type = 'transaction' } = props
+  const { chainId } = useActiveWeb3React()
+  if (!chainId) {
+    return null
+  }
+
+  const linkLabel = label || getExplorerLabel(chainId, id, type)
+  return <ExternalLink href={getExplorerOrderLink(chainId, id)}>{linkLabel}</ExternalLink>
```
Comment (W3stside) [inline src/custom/components/ExplorerLink/index.tsx:25]: nit: build the order link outside the component href

File: src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx [hunk 3]
```diff
@@ -0,0 +1,226 @@
+import { ChainId, Currency } from '@uniswap/sdk'
```
Comment (W3stside) [inline src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx:1]: what is the change here?

**Review by W3stside — APPROVED** [review]
approved with commentes

**Comment by W3stside** [comment]
build broken, probably the yarn lock

**Review by alfetopito — APPROVED** [review]
Looks good. Just the build failure as David mentioned

File: src/custom/components/AccountDetails/index.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/AccountDetails/index.tsx:211]: Not from this PR, although I think the problem is this is a Mod, that we didn't follow the pattern. Maybe I can open an issue to do the `Mod` of this component

File: src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx (hunk 3 above)
Comment (anxolin) [reply src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx:1]: image

I followed the procedure. The mod compares line by line, easy to see the change as I show in the image above. 

Also, this refers to the change in the description:

> * Adapted the confirmation modal 

Additionally, the index.tsx documents why it was created: 
image

**Pull request merged**